### PR TITLE
Rob change license

### PR DESCRIPTION
**Pull request by RobertLRead**
Braydie, here is a pull request that changes the title (slightly) and changes the license to CC BY-SA 4.0, which I think will be best for community involvement.

Let me know if you have a problem with any of this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,5 +95,8 @@ Welcome to the tribe.
         - [How to Deal with Organizational Chaos](3-Advanced/Serving-Your-Team/11-How to Deal with Organizational Chaos.md)
 4. [Glossary](4-Glossary.md)
 5. [Appendix A - Bibliography/Websiteography](5-Bibliography.md)
-6. [Appendix B - History (As of February 2003)](6-History.md)
-7. [Appendix C - GNU Free Documentation License](LICENSE)
+6. [Appendix B - History (As of January 2016)](6-History.md)
+6. [Appendix C - Contributions (As of January 2016)](7-Contributions.md)
+
```
Comment (braydie) [inline README.md]: Do you think we should continue to include a link to the license - Appendix D perhaps?

File: README.md [hunk 2]
```diff
@@ -95,5 +95,8 @@ Welcome to the tribe.
         - [How to Deal with Organizational Chaos](3-Advanced/Serving-Your-Team/11-How to Deal with Organizational Chaos.md)
 4. [Glossary](4-Glossary.md)
 5. [Appendix A - Bibliography/Websiteography](5-Bibliography.md)
-6. [Appendix B - History (As of February 2003)](6-History.md)
-7. [Appendix C - GNU Free Documentation License](LICENSE)
+6. [Appendix B - History (As of January 2016)](6-History.md)
```
Comment (braydie) [inline README.md]: As git will track changes, do you think we need to include "As of..." in the titles of these sections? It means it's another thing to update when updating history/contributions..?

**Comment by braydie** [comment]
Looks good to me! Just wondered about keeping a link to the license in the contents

**Comment by RobertLRead** [comment]
I in fact added a logo-ized licencse statement at the bottom of the README, as well as changing the LICENSE.md file.  However, I think you are correct, it should be more prominent, so I just added one at the TOP of the README as well (but without the logo.)

**Comment by RobertLRead** [comment]
messed up the link and fixed it, I think if your merge this you will have everything.  Feel free to change style of presentation further, but I assume you agree that a link at the top and bottom of the README.md is sufficient.

Someday we might want to put the logo-link code in each section, but I don't think that is necessary now.

**Pull request merged**